panel/io/server.py

```diff
@@ -723,7 +723,7 @@ def handle_exception(handler, e):
             # Serve error
             e_msg = str(e).replace('\033[1m', '<b>').replace('\033[0m', '</b>')
             Alert(
-                f'<b>{type(e).__name__}</b>: {e_msg}</br><pre style="overflow-y: auto">{tb}</pre>',
+                f'<b>{type(e).__name__}</b>: {e_msg}\n<pre style="overflow-y: auto">{tb}</pre>',
```

Comment:
I presume this assumes that `Alert` always uses preformatted text. Otherwise newlines have no bearing on text flow. Perhaps a better fix would be to use `<br>` (no `/` at all; `<br>` is self closing).

Reply:
The string is passed through a Markdown parser.

Reply:
OK, so this is not HTML, it's actually Markdown. I would somehow indicate this, because I'm pretty sure the next reader of this code will be as confused as I was.